datastore/src/test/java/io/spine/server/storage/datastore/NewBoundedContextBuilderTest.java

```diff
@@ -45,14 +43,14 @@ void testProduceBCBuilder() {
         DatastoreStorageFactory factory = givenFactory();
         BoundedContextBuilder builder = BoundedContext.multitenant(
                 NewBoundedContextBuilderTest.class.getName());
-        assertFalse(builder.tenantIndex()
-                           .isPresent());
+        assertThat(builder.tenantIndex()).isEmpty();
 
         Optional<? extends TenantIndex> updatedIndex =
                 factory.configureTenantIndex(builder)
                        .tenantIndex();
-        assertTrue(updatedIndex.isPresent());
-        assertThat(updatedIndex.get(), instanceOf(TestNamespaceIndex.getType()));
+        assertThat(updatedIndex).isPresent();
```

Comment:
Please apply the formatting as in the suggestion above — the assertion on a new line. It's our conversion. It makes assertions easier to read.